travis.sh

```diff
@@ -3,7 +3,7 @@
 set -euo pipefail
 
 function configureTravis {
-  mkdir ~/.local
+  mkdir -p ~/.local
```

Comment:
`-p` will create the directory path recursively making sure the whole path is created and also will not fail if directory is already created. It's more safe and cross machine this way. `~/.local` directory is being used by other software and in that way I can run this on my machine for example.